src/statman_app.erl

```diff
@@ -10,7 +10,26 @@
 %% ===================================================================
 
 start(_StartType, _StartArgs) ->
-    statman_sup:start_link([1000]).
+    {ok, Pid} = statman_sup:start_link([1000]),
+    case application:get_env(statman, start_aggregator, true) of
+        true ->
+            statman_aggregator_sup:add_worker(statman_aggregator);
+        false ->
+            ok
+    end,
+
+    case application:get_env(statman, start_vm_metrics, true) of
+        true ->
+            init_vm_metrics();
+        false ->
+            ok
+    end,
+    {ok, Pid}.
 
 stop(_State) ->
     ok.
+
+init_vm_metrics() ->
+    ok = statman_poller:add_gauge(fun statman_vm_metrics:get_gauges/0),
+    ok = statman_poller:add_gauge(fun statman_vm_metrics:os_stats/0),
+    ok = statman_poller:add_counter(fun statman_vm_metrics:get_counters/0).
```

Comment:
Maybe make this config-driven for flexibility? A library might not need to have os_starts reported.

Reply:
Yes, it is, there is the variable `start_vm_metrics`

Reply:
I meant a config variable that would allow specifying particular metrics that need to be reported, as currently there's no way of selecting a subset of the 3.

Reply:
Yes, but in that case you can deactivate it and start them manually in your app I guess?

Reply:
We also need to move the disk usage from our codebase to here, so we will get to improve this when we do that as well.